R/graph.R

```diff
@@ -17,14 +17,14 @@ is_referenced_data_model <- function(data_model, table_name) {
   any(which_ind)
 }
 
-#' Get the names of a [`dm`]'s tables referencing a given table.
+#' Get the names of the tables of a [`dm`]'s that reference a given table.
 #'
 #' @inheritParams cdm_is_referenced
 #'
-#' @return Character vector of the names of the tables pointing to the primary
-#' key of parameter `table`.
+#' @return Character vector of the names of the tables that point to the primary
+#' key of `table`.
 #'
-#' @family functions utilizing foreign key relations
+#' @family functions that utilize foreign key relations
```

Comment:
Note that the `@family` elements must be consistent across all files.

Reply:
```suggestion
#' @family functions utilizing foreign key relation
```